Annotation/declarative_golden_set.json

```diff
@@ -1354,7 +1452,7 @@
                     }
                 ],
                 "type": "declarative",
-                "clause": "",
+                "clause": "that its informant was responsible for first alerting law enforcement to the Twitter communications, but refused disclosure under the government\u2013informant privilege",

```

Comment:
I'm not sure I would include "but refused xxx" in this, surely the structure is "The government [agreed xx] but [refused xx"